packages/client/src/components/MemoryGameCard/MemoryGameCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { theme as t, CardProps, Card, Icon } from '@codement/ui';
+
+export interface MemoryGameCardProps extends CardProps {
+    flipped?: boolean;
+    state?: 'correct' | 'incorrect';
+    content?: string;
+}
+
+const StyledCard = styled(Card)<MemoryGameCardProps>`
+  position: relative;
+  width: 160px;
+  height: 160px;
+  left: 0%;
+  top: 0%;
+
+  background: ${t.color('white')};
+  box-shadow: 0px 2px 10px ${t.color('grey.200')};
```

Comment:
We should have a `t.shadow` available for this. If it's not configured for what we need, feel free to add it too!

Reply:
<Card /> has shadow and background setup already, so remove shadow and background from here.